core/src/main/java/org/bitcoinj/evolution/QuorumStateManager.java

```diff
@@ -0,0 +1,12 @@
+package org.bitcoinj.evolution;
+
+import java.io.FileNotFoundException;
+
+public interface QuorumStateManager {
+    void save() throws FileNotFoundException;
+    boolean isLoadedFromFile();
+    boolean notUsingBootstrapFile();
+    boolean notUsingBootstrapFileAndStream();
+    void setLoadingBootstrap();
+    void loadBootstrapAndSync();
+}
```

Comment:
This interface allows an `AbstractQuorumState` or derived class to query what is happening with loading and saving files.

`SimplifiedMasternodeListManager` has a single file to store the masternode/quorum state, but there are two systems (`QuorumState` and `QuorumRotationState`) that will need to be saved into that file.